test.sh

```diff
@@ -1,4 +1,17 @@
 #!/bin/bash
+
+if [ ! -f media/extension.js ]; then
+	printf "Running webpack\n"
+	webpack --env.development
+fi
```

Comment:
:point_up: This bit will be useful when we run in CI.

Fun fact: the tests are loaded from the `tsc`-compiled JavaScript emitted by this script, but the development extension that's loaded into the extension test host uses `media/extension.ts`, so they both need to exist.